gesso_helper/src/ThemeSettings.php

```
@@ -0,0 +1,56 @@
+   * @return mixed
+   *   The setting value, or NULL if not set.
+   */
+  public function getSetting(string $setting_name, ?string $theme = NULL): mixed {
+    if ($this->themeSettingsProvider !== NULL) {
+      return $this->themeSettingsProvider->getSetting($setting_name, $theme);
+    }
+
+    // Drupal 10.3–11.2 fallback. Removed in Drupal 13;
+    // provider path is preferred.
+    // @phpstan-ignore function.deprecated
+    return theme_get_setting($setting_name, $theme);
```

PHPStan (2.2.6)

[error] No error with identifier function.deprecated is reported on line 53.


(ignore.unmatchedIdentifier)